example/lib/src/ui/device_detail_screen.dart

```diff
@@ -1,193 +1,106 @@
-import 'dart:async';
-import 'dart:convert';
-
 import 'package:flutter/material.dart';
 import 'package:flutter_reactive_ble/flutter_reactive_ble.dart';
+import 'package:flutter_reactive_ble_example/src/ble/ble_device_connector.dart';
+import 'package:provider/provider.dart';
 
-import '../utils.dart';
-import '../widgets.dart';
-
-final _serviceUuid = Uuid.parse("932C32BD-0000-47A2-835A-A8D455B859DD");
-final _charUuid = Uuid.parse("932C32BD-0002-47A2-835A-A8D455B859DD");
-
-class DeviceDetailScreen extends StatefulWidget {
+class DeviceDetailScreen extends StatelessWidget {
```

Comment:
This screen is very basic now it has a connect and disconnect button and displays the connect state + device name and ID. Prefer to show a limited set and a proper architecture to our users on how to implement detail screen